edweiss-app/__test__/components/lectures/remotecontrol/modal.test.tsx

```diff
@@ -457,3 +468,106 @@ describe('QuestionBroadcastModal', () => {
     });
 });
 
+
+// ------------------------------------------------------------
+// -------      Quiz Broadcast Modal Test Suites       --------
+// ------------------------------------------------------------
+
+describe('QuizBroadcastModal', () => {
+    const mockOnClose = jest.fn();
+    const mockSetBroadcasted = jest.fn();
+    const mockModalRef = { current: null };
+
+    const quizModel = {
+        answer: {
+            type: "TFAnswer",
+            value: true,
+        },
+        ended: false,
+        exercise: {
+            answer: true,
+            question: "Is earth flat ?",
+            type: "TF",
+        },
+        showResultToStudents: false,
+    } as LectureQuizzes.LectureQuiz;
+
+    const renderModal = (props = {}) =>
+        render(
+            <QuizBroadcastModal
+                modalRef={mockModalRef}
+                id="1"
+                courseId="course1"
+                lectureId="lecture1"
+                quizModel={quizModel}
+                broadcasted=""
+                setBroadcasted={mockSetBroadcasted}
+                onClose={mockOnClose}
+                {...props}
+            />);
+
+
+    afterEach(() => {
+        jest.clearAllMocks();
```

Comment:
I personally prefer doing a jest.clearAllMocks(); as a beforeEach instead of afterEach since I think it is less prone to errors